### DISCUSSION TITLE
SCIM Support

### DISCUSSION THREAD

**Discussion by cekageka** [GitbookIO/community#449] [Feature Requests]
SSO in Gitbook is a good start, but it's a nightmare in complexity and security to have distributed group management and in many cases the need to manually duplicate groups in Gitbook which are already maintained in the primary identity system (Entra, Google, whatever).

In modern SaaS the obvious solution is SCIM. I know you're a small team, but I know from SaaS software maintained by single developers successfully implemented SCIM. It would be great if you would implement it, too.

There was a section in your doc about SCIM, but I it's only visible with Waybackmachine. Hope you've not taken it away from your roadmap: https://web.archive.org/web/20230923064408/https://docs.gitbook.com/product-tour/sso-and-saml/scim-and-directory-sync

Of course, a custom code against the Gitbook API would also be suitable to enable the sync. But SCIM was invented precisely to avoid this 'individual coding against each individual SaaS system'. 

<img width="1167" alt="scim" src="https://github.com/GitbookIO/community/assets/1116271/d231dad2-92a5-4374-aac5-1380d41a69f3">

**Comment by kitforbes** [comment]
We've just configured SSO with Entra and I was looking for group mappings as the final step. Hopefully this will be addressed one day!
